[*] Start mutation process:
   - targets: fizz_buzz.py
   - tests: ts-3-5/test_3-5_0-8_263.py
[*] 5 tests passed:
   - test_3-5_0-8_263 [0.12545 s]
[*] Start mutants generation and execution:
   - [#   1] AOR fizz_buzz: 
--------------------------------------------------------------------------------
  5:     divisible by 5, add "Buzz", if it is 
  6:     divisible by both, add "FizzBuzz"
  7: 
  8:     '''
- 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
+ 9:     return [(('Fizz' if x * 3 == 0 else '') + ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
--------------------------------------------------------------------------------
[0.14067 s] killed by ts-3-5/test_3-5_0-8_263.py::test_fizz_buzz_divisible_by_3

def test_fizz_buzz_divisible_by_3():
        result = fizz_buzz(9)
>       assert result[2] == "Fizz"
E       AssertionError

ts-3-5/test_3-5_0-8_263.py:11: AssertionError
   - [#   2] AOR fizz_buzz: 
--------------------------------------------------------------------------------
  5:     divisible by 5, add "Buzz", if it is 
  6:     divisible by both, add "FizzBuzz"
  7: 
  8:     '''
- 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
+ 9:     return [(('Fizz' if x % 3 == 0 else '') - ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
--------------------------------------------------------------------------------
[0.14079 s] incompetent

TypeError: fizz_buzz:9: TypeError: unsupported operand type(s) for -: 'str' and 'str'

   - [#   3] AOR fizz_buzz: 
--------------------------------------------------------------------------------
  5:     divisible by 5, add "Buzz", if it is 
  6:     divisible by both, add "FizzBuzz"
  7: 
  8:     '''
- 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
+ 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x * 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
--------------------------------------------------------------------------------
[0.13880 s] killed by ts-3-5/test_3-5_0-8_263.py::test_fizz_buzz_divisible_by_5

def test_fizz_buzz_divisible_by_5():
        result = fizz_buzz(10)
>       assert result[4] == "Buzz"
E       AssertionError

ts-3-5/test_3-5_0-8_263.py:17: AssertionError
   - [#   4] AOR fizz_buzz: 
--------------------------------------------------------------------------------
  5:     divisible by 5, add "Buzz", if it is 
  6:     divisible by both, add "FizzBuzz"
  7: 
  8:     '''
- 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
+ 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n - 1)]
--------------------------------------------------------------------------------
[0.14294 s] killed by ts-3-5/test_3-5_0-8_263.py::test_fizz_buzz_divisible_by_3

def test_fizz_buzz_divisible_by_3():
        result = fizz_buzz(9)
        assert result[2] == "Fizz"
        assert result[5] == "Fizz"
>       assert result[8] == "Fizz"
E       IndexError: list index out of range

ts-3-5/test_3-5_0-8_263.py:13: IndexError
   - [#   5] LCR fizz_buzz: 
--------------------------------------------------------------------------------
  5:     divisible by 5, add "Buzz", if it is 
  6:     divisible by both, add "FizzBuzz"
  7: 
  8:     '''
- 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
+ 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x % 5 == 0 else '') and str(x)) for x in range(1, n + 1)]
--------------------------------------------------------------------------------
[0.13778 s] killed by ts-3-5/test_3-5_0-8_263.py::test_fizz_buzz_divisible_by_3

def test_fizz_buzz_divisible_by_3():
        result = fizz_buzz(9)
>       assert result[2] == "Fizz"
E       AssertionError

ts-3-5/test_3-5_0-8_263.py:11: AssertionError
   - [#   6] ROR fizz_buzz: 
--------------------------------------------------------------------------------
  5:     divisible by 5, add "Buzz", if it is 
  6:     divisible by both, add "FizzBuzz"
  7: 
  8:     '''
- 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
+ 9:     return [(('Fizz' if x % 3 != 0 else '') + ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
--------------------------------------------------------------------------------
[0.13714 s] killed by ts-3-5/test_3-5_0-8_263.py::test_fizz_buzz_divisible_by_3

def test_fizz_buzz_divisible_by_3():
        result = fizz_buzz(9)
>       assert result[2] == "Fizz"
E       AssertionError

ts-3-5/test_3-5_0-8_263.py:11: AssertionError
   - [#   7] ROR fizz_buzz: 
--------------------------------------------------------------------------------
  5:     divisible by 5, add "Buzz", if it is 
  6:     divisible by both, add "FizzBuzz"
  7: 
  8:     '''
- 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
+ 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x % 5 != 0 else '') or str(x)) for x in range(1, n + 1)]
--------------------------------------------------------------------------------
[0.13843 s] killed by ts-3-5/test_3-5_0-8_263.py::test_fizz_buzz_divisible_by_3

def test_fizz_buzz_divisible_by_3():
        result = fizz_buzz(9)
>       assert result[2] == "Fizz"
E       AssertionError

ts-3-5/test_3-5_0-8_263.py:11: AssertionError
   - [#   8] SDL fizz_buzz: 
--------------------------------------------------------------------------------
  5:     divisible by 5, add "Buzz", if it is 
  6:     divisible by both, add "FizzBuzz"
  7: 
  8:     '''
- 9:     return [(('Fizz' if x % 3 == 0 else '') + ('Buzz' if x % 5 == 0 else '') or str(x)) for x in range(1, n + 1)]
+ 9:     pass
--------------------------------------------------------------------------------
[0.14777 s] incompetent

TypeError: /home/lucca/teste/python_experiments2/fizz-buzz/ts-3-5/test_3-5_0-8_263.py:11: TypeError: 'NoneType' object is not subscriptable

[*] Mutation score [1.34670 s]: 100.0%
   - all: 8
   - killed: 6 (75.0%)
   - survived: 0 (0.0%)
   - incompetent: 2 (25.0%)
   - timeout: 0 (0.0%)
